src/App.js

```diff
@@ -2,12 +2,72 @@ import React from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  showAll = () => {
+    getAll().then((data) => {
+      this.setState({ goods: data });
+    });
+  }
+
+  show5First = () => {
+    get5First().then((data) => {
+      this.setState({ goods: data });
+    });
+  }
+
+  showRed = () => {
+    getRedGoods().then((data) => {
+      this.setState({ goods: data });
+    });
+  }
```

Comment:
you could make 1 function instead of these 3 and pass callback as a parameter to it